### ISSUE TITLE
How about to change circle-map to C++ version?

### ISSUE THREAD

**Issue by panxiaoguang** [nf-core/circdna#66]
### Description of feature

The original version of Circle-Map is a little bit slow, especially when it processes reads that contain a high percentage of ChrM. The C++ reimplementation of Circle-Map can deal with this situation and will be a faster alternative. Thus, It would be a good idea to use C++ version in the pipeline.

**Comment by panxiaoguang** [comment]
please find it in https://github.com/BGI-Qingdao/Circle-Map-cpp

**Comment by DSchreyer** [comment]
I agree. Can you try to make a pull request to dev for this ? I might be able to include it then into the newest version 1.1
